nuplan_garage/planning/simulation/planner/pdm_planner/pdm_closed_planner.py

```diff
@@ -6,50 +6,54 @@
 
 from nuplan.planning.simulation.observation.observation_type import Observation
 from nuplan.planning.simulation.planner.abstract_planner import PlannerInitialization, PlannerInput
-from nuplan_garage.planning.simulation.planner.pdm_planner.abstract_pdm_planner import AbstractPDMPlanner
-from nuplan_garage.planning.simulation.planner.pdm_planner.proposal.batch_idm_policy import BatchIDMPolicy
-from nuplan_garage.planning.simulation.planner.pdm_planner.utils.route_utils import (
-    route_roadblock_correction,
-)
 from nuplan.planning.simulation.trajectory.abstract_trajectory import AbstractTrajectory
+from nuplan.planning.simulation.trajectory.trajectory_sampling import TrajectorySampling
+
+from nuplan_garage.planning.simulation.planner.pdm_planner.abstract_pdm_closed_planner import (
+    AbstractPDMClosedPlanner,
+)
+from nuplan_garage.planning.simulation.planner.pdm_planner.observation.pdm_observation_utils import (
+    get_drivable_area_map,
+)
+from nuplan_garage.planning.simulation.planner.pdm_planner.proposal.batch_idm_policy import (
+    BatchIDMPolicy,
+)
 
 
 import warnings
-warnings.filterwarnings("ignore", category=RuntimeWarning) 
+
+warnings.filterwarnings("ignore", category=RuntimeWarning)
```

Comment:
Which warning does this filter? can we be more specific here to reduce FP of the filter?